python/GafferImageUI/CatalogueUI.py

```diff
@@ -1096,6 +1119,31 @@ def __keyPress( self, imageListing, keyEvent ) :
 
 		return False
 
+	def __buttonPress( self, pathListing, event ) :
+
+		cellColumn = pathListing.columnAt( event.line.p0 )
+		if not isinstance( cellColumn, OutputIndexColumn ):
+			return False
+
+		return True
+
+	def __buttonRelease( self, pathListing, event ) :
+
+		cellColumn = pathListing.columnAt( event.line.p0 )
+		if not isinstance( cellColumn, OutputIndexColumn ):
+			return False
+
+		if event.button == event.Buttons.Left and event.modifiers == event.Modifiers.None_ :
+			name = pathListing.pathAt( event.line.p0 )
+			if name:
+				image = _imageFromName( self.getPlug().node(), name[0] )
+				self.__setComparisonImage( image["outputIndex"].getValue() == 0, image )
+
+		return True
+
+	def __setComparisonImage( self, index, image, *unused ) :
+		image["outputIndex"].setValue( index )
```

Comment:
I guess this is deliberately not wrapped in an UndoScope? Because we think the output index stuff should feel like the selection, and that isn't undoable either?